docs/dev_guide/update_python_docs.md

```diff
@@ -9,20 +9,20 @@ parent: Feathr Developer Guides
 
 `pip install sphinx &&  pip install sphinx_rtd_theme`
 
-Since `sphinx` need to run the source code to generate the documentation, make sure you have installed the all the necessary dependencies needed by the project(see setup.py files).
+Since `sphinx` need to run the source code to generate the documentation, make sure you have installed all the necessary dependencies needed by the project(see setup.py files).
 
 ## How Readthedocs.com Works
 * Readthedocs need to run the Python code to generate the docs. So your dependencies need to exist on the machine that generates these docs.
 * Readthedocs can be configured via `.readthedocs.yml` under project root(not your Python library root). If you don't 
   need extra configurations, you don't need `.readthedocs.yml`.
 * Readthedocs will generate documentations based on your `.rst` file. `.rst` file can be automatically generated or manually modified(see later sections).
-* Readthedocs doesn't support complex dependencies, like Pypi packages with C modules. To address this, Readthedocs provides those common
+* Readthedocs doesn't support complex dependencies, like PyPI packages with C modules. To address this, Readthedocs provide those common
   dependencies in their server but we need to enable it by setting `system_packages: true`
 * Dependencies can be specified by docs/requirements.txt file.
 * You don't need to commit _build files. Those files will be generated by the Readthedocs.com servers.
 
 ### How to Specify Dependencies
-Readthedocs provides two ways of specifying depencies.
+Readthedocs provide two ways of specifying dependencies.
```

Comment:
nit: Readthedocs should be singular, like Facebook, LinkedIn, even though it ends with `s`. If you know any source that mention companies or products end with `s` should be considered plural. Let me know.

Reply:
@hangfei 
Ok done! Reverted back to singular. Thanks for the heads up!